swiftui/ChartboostMediationDemo/Ads/Fullscreen/FullscreenAdView.swift

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022-2023 Chartboost, Inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file.
+
+//
+//  InterstitialAdView.swift
+//  ChartboostMediationDemo
+//
+//  Copyright © 2023 Chartboost. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+/// A view that demonstrates the loading and showing of a Chartboost Mediation SDK interstitial advertisement.
+struct FullscreenAdView: View {
+    @StateObject private var controller: FullscreenAdController
+    @State private var isBusy = false
+    @State private var failureMessage: String?
+
+    var body: some View {
+        VStack {
+            Image("Interstitial")
+                .padding(.vertical, 32)
+
+            Text("A full screen interstitial advertisement must first be loaded.")
```

Comment:
should we add another dependency to this view named "format" or something like that, and use it to differentiate between "interstitial" and "rewarded"?
This new view is supposed to serve for both formats but now it seems to be hardcoded to display interstitial info only